commodore/postprocess/jsonnet.py

```diff
@@ -115,15 +116,14 @@ def _inventory() -> Dict[str, Any]:
             yaml_dump(outcontents, outpath)
 
 
-def _filter_file(work_dir: P, component: str, filterpath: str) -> P:
-    # TODO: Do we need to handle search path better?
-    return work_dir / "dependencies" / component / filterpath
+def _filter_file(component: Component, filterpath: str) -> P:
+    return component.target_directory / filterpath
```

Comment:
The PR diff is a bit noisy because I had to change the whole postprocessing infrastructure to work with the `Component` object instead of the component name, so that this change is possible.